spec/views/users_index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.feature 'UsersIndices', type: :feature do
+  let!(:user) { User.find(1) }
+
+  def image_exists?(url)
+    response = Net::HTTP.get_response(URI.parse(url))
+    response.code == '200'
+  rescue StandardError
+    false
+  end
+
+  it 'displays the user name and profile image on the index page' do
```

Comment:
- [ ] Once more, let's kindly make sure all our test cases are running. Currently, this particular test is still failing as you can see in the screenshot below:


![Screenshot from 2023-04-20 11-19-42](https://user-images.githubusercontent.com/63087491/233338456-595ba109-89a9-4c94-baa5-2ba3eb5a9696.png)

Reply:
Hi @FrederickMih,

As explained on Slack....all the tests are passing on my terminal as seen below....I don't know why it's failing on your end.
![Code_smg1JbXMSX](https://user-images.githubusercontent.com/106531695/233344824-858efa03-69b8-44a7-87e4-f2b01c2e000b.png)
.

Reply:
I have updated the test and also done a test...all the tests are passing as per the screenshot below. 
![Code_0Sa1wwGKys](https://user-images.githubusercontent.com/106531695/233348581-a8b7eedb-d328-4983-8279-59a19f02f29c.png)